### DISCUSSION TITLE
[BUG] NameError: uninitialized constant Paddle::VERSION

### DISCUSSION THREAD

**Discussion by ovidiuanca** [pay-rails/pay#915] [Q&A] [answered]
## Bug Report

**Describe the Bug:**
After upgrading to pay v7.0.0, I am getting the following error when running my test suite.

```
NameError:
  uninitialized constant Paddle::VERSION
# /Users/_/.rvm/gems/ruby-3.2.2/gems/pay-7.0.0/lib/pay/paddle_billing.rb:19:in `enabled?'
```

**To Reproduce:**

1. Upgrade `pay` to version `7.0.0`.
2. Run tests or zeitwerk check.

**Environment:**
- Pay gem version: 7.0.0
- Ruby version: 3.2.2
- Rails version: 7.1.2
- Operating System: MacOS 14.1.2

**Comment by excid3** [comment, marked as answer]
Did you follow the upgrade guide and change Paddle gems?

**Comment by ovidiuanca** [comment]
Totally missed the notes in `UPGRADE.md`. 🤦 

All is good after following the steps. 

Thanks for the quick response.

**Reply by excid3** [reply]
Haha! Major version bump for a reason 😜
